src/elements/toniq-dropdown/toniq-dropdown.element.ts

```diff
@@ -0,0 +1,188 @@
+    },
+    styles: css`
+        :host {
+            font-family: var(--toniq-font, 'Rubik');
+            font-style: normal;
+            font-weight: 700;
+            font-size: 16px;
+            line-height: 24px;
+        }
+
+        .dropdown {
+            width: 288px;
+            height: 48px;
+            position: relative;
+        }
+
+        toniq-icon {
+            -moz-transition: all 0.2s linear;
+            -webkit-transition: all 0.2s linear;
+            transition: all 0.2s linear;
+        }
+
+        .dropdown.open toniq-icon {
+            -moz-transform: rotate(180deg);
+            -webkit-transform: rotate(180deg);
+            transform: rotate(180deg);
+        }
+
+        .dropdown.open .select-options {
+            display: grid;
+        }
+
+        .select {
+            display: flex;
+            justify-content: space-between;
+            border-radius: 8px;
+            cursor: pointer;
+            padding: 12px 12px 12px 16px;
+            -webkit-touch-callout: none; /* iOS Safari */
+            -webkit-user-select: none; /* Safari */
+            user-select: none; /* Non-prefixed version, currently supported by Chrome, Edge, Opera and Firefox */
+            ${applyBackgroundAndForeground(toniqColors.accentSecondary)}
+        }
+
+        .select-options {
+            display: none;
+            position: absolute;
+            top: 100%;
+            left: 0;
+            right: 0;
+            z-index: 99;
+            border-radius: 8px;
+            ${applyBackgroundAndForeground(toniqColors.pagePrimary)}
+            filter: drop-shadow(0px 16px 16px rgba(39, 170, 243, 0.16));
+        }
+
+        .select-options .option {
+            padding: 16px 16px 8px 16px;
+            cursor: pointer;
+            user-select: none;
+        }
+
+        .select-options .option.selected,
+        .option:hover {
+            ${applyBackgroundAndForeground(toniqColors.accentPrimary)}
+        }
+
+        .select-options .option:last-child {
+            padding: 16px 16px 16px 16px;
+            border-radius: 0px 0px 8px 8px;
+        }
+    `,
+    renderCallback: ({dispatch, events, props, setProps, host}) => {
+        const defaultSelectProps: Required<ToniqDropdownSelect> = {
+            value: props.list?.length ? props.list[0]?.value : '',
+            label: props.list?.length ? props.list[0]?.label : '',
+        };
+
+        const list = props.list ?? [];
+        const select =
+            typeof props.select === 'object' && Object.keys(props.select).length
+                ? props.select
+                : defaultSelectProps;
+        let dropDownEl: HTMLElement;
+
+        // Get dropdown element after rendering
+        setTimeout(() => {
+            dropDownEl = host.shadowRoot?.querySelector('div.dropdown') as HTMLElement;
+            new ToniqDropdownController(host, dropDownEl);
+        }, 500);
+
+        const onToggleDropdown = () => {
```

Comment:
`onToggleDropdown` and `onSelectOption` can just use function notation because they aren't accessing `this`:
```suggestion
        function onToggleDropdown() {
```